src/Utility/ExprFunctions.h

```diff
@@ -102,7 +102,7 @@ class EvalFunction : public RealFunc
 
   std::vector<Arg> arg; //!< Function argument values
 
-  std::array<std::unique_ptr<EvalFunction>,4> gradient;  //!< First derivative expressions
+  std::array<std::unique_ptr<EvalFunction>,4> fgradient;  //!< First derivative expressions
   std::array<std::unique_ptr<EvalFunction>,6> dgradient; //!< Second derivative expressions
```

Comment:
OK, I see. You changed then name only because you introduced a new method with that name. Then my suggestion will not work, but still I feel fgradient and dgradient are not optimal variable names here.

Reply:
'derivatives' and 'hessian' then maybe?

Reply:
well, you seem to introduce the `hessian` as a method in the next (#640)...

Reply:
doh, yes. uhm, then i'm not sure what else to name them unless we add m_ or trailing _ or something.

Reply:
Maybe `derivative1` and `derivative2` is as good as anything else

Reply:
yes, let's do that.